docker-compose.yml

```diff
@@ -1,24 +1,49 @@
 version: '3'
 services:
-  web:
-    build: .
+
+  api:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: mysterium-api
     ports:
-      - "5000:5000"
+      - 8001:5000
+    environment:
+      DB_HOST: mysterium-mysql
+      DB_NAME: myst_api
+      DB_USER: myst_api
+      DB_PASSWORD: myst_api
+    volumes:
+      - .:/code
```

Comment:
Our servers do not support hot-reloading yet, so there is no point in bounding volume.